custom-recipes/timeseries-preparation-resampling/recipe.py

```diff
@@ -12,7 +14,10 @@
 recipe_config = get_recipe_config()
 datetime_column = recipe_config.get('datetime_column')
 if recipe_config.get('advanced_activated') and recipe_config.get('groupby_column'):
+    logger.warning("The field 'Column with identifier' is deprecated. Please remove the current value and use the field 'Column with identifiers' instead")
     groupby_columns = [recipe_config.get('groupby_column')]
+elif recipe_config.get('advanced_activated') and recipe_config.get('groupby_columns'):
+    groupby_columns = recipe_config.get('groupby_columns')
 else:
     groupby_columns = None
```

Comment:
because this code will be used in all the recipe, maybe you could put it in a function in the python-lib

Reply:
done in https://github.com/dataiku/dss-plugin-timeseries-preparation/pull/16/commits/860de22943dc57fabd1c6c30213214dbacdaa026